cmd/block-generator/main.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"math/rand"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/spf13/viper"
+
+	"github.com/algorand/indexer/cmd/block-generator/generator"
+	"github.com/algorand/indexer/util"
+)
+
+var configFile string
+var port uint
+var gen generator.Generator
+
+const configFileName = "block_generator_config"
+
+func init() {
+	rand.Seed(12345)
+	flag.StringVar(&configFile, "config", "", "Override default config file.")
+	flag.UintVar(&port, "port", 4010, "Port to start the server at.")
+
+	viper.SetConfigName(configFileName)
+	viper.SetConfigType("yaml")
+	viper.AddConfigPath(".")
+}
+
+func initializeConfigFile() error {
+	var err error
+	if len(configFile) > 0 {
+		var f *os.File
+		f, err = os.Open(configFile)
+		if err != nil {
+			return err
+		}
+
+		return viper.ReadConfig(f)
+	}
+
+	return viper.ReadInConfig()
```

Comment:
could you add the default config name in the help text / flag description?

Reply:
done